ocs_ci/ocs/platform_nodes.py

```diff
@@ -2186,3 +2186,50 @@ class RHVNodes(NodesBase):
     def __init__(self):
         super(RHVNodes, self).__init__()
         self.rhv = rhv.RHV()
+
+    def stop_nodes(self, nodes, timeout=600, wait=True, force=True):
+        """
+        Shutdown RHV VM
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the instances to stop, False otherwise
+            timeout (int): time in seconds to wait for node to reach 'not ready' state.
+            force (bool): True for force VM stop, False otherwise
+
+        """
+        if not nodes:
+            raise ValueError("No nodes found to stop")
+
+        node_names = [n.name for n in nodes]
+
+        self.rhv.stop_rhv_vms(node_names, force=force)
```

Comment:
force=force doesn't make much sense, probably best is to fix with `force=true`

Reply:
`force` is initialized with `True` as the default value in [#L2215](https://github.com/red-hat-storage/ocs-ci/blob/3fc889169aa0ede3d6fc82e237c47a81e4e6d9df/ocs_ci/ocs/platform_nodes.py#L2215)
Ans received `force` argument value must pass to the wrapper function `stop_rhv_vms`